src/main/java/eu/dzhw/fdz/metadatamanagement/projectmanagement/service/DaraService.java

```diff
@@ -297,6 +297,15 @@ private Map<String, Object> getDataForDataPackageTemplate(DataAcquisitionProject
       availabilityControlled = AVAILABILITY_CONTROLLED_DELIVERY;
     }
 
+    if (project.getRelease().getDoiPageLanguage() != null && project.getRelease().getDoiPageLanguage().equals("de")) {
+      dataForTemplate.put("projectURL", "https://metadata.fdz.dzhw.eu/de/data-packages/" + dataPackage.getMasterId()
+          + "?version=" + project.getRelease().getVersion());
+    } else {
+      dataForTemplate.put("projectURL", "https://metadata.fdz.dzhw.eu/en/data-packages/" + dataPackage.getMasterId()
+          + "?version=" + project.getRelease().getVersion());
+    }
```

Comment:
This works well enough. However I think this implementation is susceptible to follow-up mistakes. If we're going to introduce more variables at a later point that need to be part of the URL we need to make sure that they are also included in in this if-else-block. For that reason I propose we introduce a template variable for the landing page language as well and simply replace it instead of the entire URL. Unless there is another issue with this approach that I'm not seeing.